pkg/network/tracer/tracer.go

```diff
@@ -637,6 +640,38 @@ func (t *Tracer) getConnTelemetry(mapSize int) *network.ConnectionsTelemetry {
 	return tm
 }
 
+func (t *Tracer) getRuntimeCompilationTelemetry() map[string]network.RuntimeCompilationTelemetry {
+	compilationTelemetryByAsset := make(map[string]network.RuntimeCompilationTelemetry)
+
+	ttm := network.RuntimeCompilationTelemetry{}
+	tracerTelemetry := runtime.Tracer.GetTelemetry()
+	if enabled, ok := tracerTelemetry["runtime_compilation_enabled"]; ok {
+		ttm.RuntimeCompilationEnabled = enabled == 1
+	}
+	if result, ok := tracerTelemetry["runtime_compilation_result"]; ok {
+		ttm.RuntimeCompilationResult = int32(result)
+	}
+	if duration, ok := tracerTelemetry["runtime_compilation_duration"]; ok {
+		ttm.RuntimeCompilationDuration = duration
+	}
+	compilationTelemetryByAsset["tracer"] = ttm
+
+	ctm := network.RuntimeCompilationTelemetry{}
+	conntrackTelemetry := runtime.Conntrack.GetTelemetry()
+	if enabled, ok := conntrackTelemetry["runtime_compilation_enabled"]; ok {
+		ctm.RuntimeCompilationEnabled = enabled == 1
+	}
+	if result, ok := conntrackTelemetry["runtime_compilation_result"]; ok {
+		ctm.RuntimeCompilationResult = int32(result)
+	}
+	if duration, ok := conntrackTelemetry["runtime_compilation_duration"]; ok {
+		ctm.RuntimeCompilationDuration = duration
+	}
+	compilationTelemetryByAsset["conntracker"] = ctm
+
+	return compilationTelemetryByAsset
```

Comment:
Can you do a loop here to loop through the two assets' telemetry instead?

Reply:
Definitely :)